plugins/common/DSP/DistortionSuperModeler.hpp

```diff
@@ -0,0 +1,301 @@
+    setting.setFloatValue(700.0);
+    addSetting(setting);
+
+    setting.init(
+        "treble",                 // Name
+        EffectSetting::floatType, // Type of setting
+        -30.0,                    // Min value
+        30.0,                     // Max value
+        0.5,                      // Step Size
+        EffectSetting::dB);
+    setting.setFloatValue(0.0);
+    addSetting(setting);
+
+    setting.init(
+        "trebleFreq",             // Name
+        EffectSetting::floatType, // Type of setting
+        1000,                     // Min value
+        7000,                     // Max value
+        0.5,                      // Step Size
+        EffectSetting::linear);
+    setting.setFloatValue(4000.0);
+    addSetting(setting);
+
+    setting.init(
+        "level",                  // Name
+        EffectSetting::floatType, // Type of setting
+        -60.0,                    // Min value
+        6.0,                      // Max value
+        0.5,                      // Step Size
+        EffectSetting::dB);
+    setting.setFloatValue(-12.0);
+    addSetting(setting);
+
+    setting.init(
+        "dryLevel",               // Name
+        EffectSetting::floatType, // Type of setting
+        -100.0,                   // Min value
+        24,                       // Max value
+        1,                        // Step Size
+        EffectSetting::dB);
+    setting.setFloatValue(-100.0);
+    addSetting(setting);
+
+    loadFromConfig();
+  };
+
+  void loadFromConfig() override
+  {
+    // Read the settings from the map and apply them to our copy of the data.
+
+    m_hpf1Freq = getSettingByName("hpf1Freq").getFloatValue();
+    m_lpf1Freq = getSettingByName("lpf1Freq").getFloatValue();
+    m_gain1 = getSettingByName("drive1").getFloatValue();
+    m_clip1Type = (SignalBlock::ClipType)getSettingByName("clip1Type").getIntValue();
+    m_hpf2Freq = getSettingByName("hpf2Freq").getFloatValue();
+    m_lpf2Freq = getSettingByName("lpf2Freq").getFloatValue();
+    m_gain2 = getSettingByName("drive2").getFloatValue();
+    m_clip2Type = (SignalBlock::ClipType)getSettingByName("clip2Type").getIntValue();
+    m_hpf3Freq = getSettingByName("hpf3Freq").getFloatValue();
+    m_lpf3Freq = getSettingByName("lpf3Freq").getFloatValue();  
+    m_toneBassCutBoost = getSettingByName("bass").getFloatValue();
+    m_toneBassFreq = getSettingByName("bassFreq").getFloatValue();
+    m_toneMidrangeCutBoost = getSettingByName("mid").getFloatValue();
+    m_toneMidrangeFreq = getSettingByName("midFreq").getFloatValue();
+    m_toneTrebleCutBoost = getSettingByName("treble").getFloatValue();
+    m_toneTrebleFreq = getSettingByName("trebleFreq").getFloatValue();
+    m_level = getSettingByName("level").getFloatValue();
+    m_dryLevel = getSettingByName("dryLevel").getFloatValue();
+
+    setupFilters();
+
+    DistortionModelerBase::loadFromConfig();
+
+
+  }
+
+private:
+  BiQuadFilter m_hpf1; // pre-clipping high-pass filter
```

Comment:
I don't know why you are declaring all of these member variables in the derived class.  They already belong to this class as part of the bass class.  So when you refer to m_hpf1 I don't know how the compiler will know if it should use the on in the derived class or the bass class.

If you need to access these variables in the derived class (like in the above function where you are setting them) you need to define them as "protected" in the base class so you can access them in the derived class.

You for sure should not declare them again here.